tests/scenarios/transform-stpf.spec.ts

```diff
@@ -238,177 +238,177 @@ describe('State proof transaction', () => {
               ],
-              "tree-depth": 10,
+              "treeDepth": 10,
             },
-            "positions-to-reveal": [
-              2,
-              13,
-              10,
-              17,
-              15,
-              5,
-              4,
-              13,
-              27,
-              15,
-              12,
-              3,
-              1,
-              15,
-              4,
-              5,
-              25,
-              21,
-              24,
-              6,
-              36,
-              7,
-              2,
-              2,
-              6,
-              19,
-              0,
-              9,
-              19,
-              9,
-              15,
-              23,
-              9,
-              8,
-              23,
-              0,
-              4,
-              4,
-              31,
-              20,
-              24,
-              1,
-              17,
-              4,
-              31,
-              15,
-              18,
-              4,
-              9,
-              26,
-              19,
-              0,
-              27,
-              20,
-              10,
-              19,
-              4,
-              3,
-              12,
-              24,
-              8,
-              14,
-              28,
-              6,
-              14,
-              12,
-              21,
-              7,
-              13,
-              20,
-              26,
-              14,
-              18,
-              10,
-              1,
-              23,
-              2,
-              39,
-              25,
-              19,
-              10,
-              18,
-              16,
-              10,
-              16,
-              22,
-              18,
-              8,
-              32,
-              7,
-              2,
-              24,
-              3,
-              24,
-              4,
-              15,
-              42,
-              19,
-              33,
-              26,
-              0,
-              4,
-              10,
-              8,
-              15,
-              0,
-              20,
-              17,
-              10,
-              13,
-              12,
-              19,
-              3,
-              11,
-              3,
-              1,
-              14,
-              3,
-              3,
-              23,
-              30,
-              5,
-              4,
-              21,
-              8,
-              10,
-              7,
-              28,
-              18,
-              14,
-              24,
-              12,
-              4,
-              9,
-              30,
-              17,
-              9,
-              32,
-              21,
-              6,
-              2,
-              0,
-              14,
-              1,
-              9,
-              7,
-              93,
-              15,
-              20,
+            "positionsToReveal": [
```

Comment:
Is this definitely a 64-bit int in core protocol?

Reply:
The reason for this change is that `algosdk` now has a type for `StateProof`. In the past, we just needed to past the stateProof in as Uint8Array, now we have to map it to a proper object. Because of this mapping, the type was changed from number to bigint (`algosdk` expects the type to be bigint)